custom/icds_reports/ucr/reports/mobile_mpr_2a_deaths.json

```diff
@@ -18,93 +18,79 @@
     ],
     "filters": [
       {
-        "compare_as_string": false,
-        "datatype": "date",
-        "required": false,
-        "display": "Date of Death",
+        "slug": "date_death_exists",
+        "type": "pre",
         "field": "date_death",
+        "pre_operator": "!=",
+        "pre_value": "",
+        "datatype": "date"
```

Comment:
I do like that better, but it looks like that's interpreted as not existing:


> Problem with report spec: Problem creating report filter from spec: { "pre_value": null, "datatype": "date", "pre_operator": "is not", "display": null, "field": "date_death", "type": "pre", "slug": "date_death_exists" }, message is: Property pre_value is required.

Reply:
This does work as-is, actually - I've been testing here: https://india.commcarehq.org/a/icds-cas/reports/configurable/534c0dab56457dd79ba47167e0f36ff9/
Feel free to remove the `pre` filter if you want to play around with it, but before adding that in, there were a bunch of rows where the date of death value was blank.

Is your concern more that this is perhaps taking advantage of an implementation quirk rather than something we expect to work?